DQM/HLTEvF/plugins/TriggerRatesMonitor.cc

```diff
@@ -339,31 +352,34 @@ void TriggerRatesMonitor::analyze(edm::Event const & event, edm::EventSetup cons
   // monitor the HLT triggers and datsets rates
   if (m_hltConfig.inited()) {
     edm::TriggerResults const & hltResults = get<edm::TriggerResults>(event, m_hlt_results);
-    if (hltResults.size() == m_hlt_counts.size()) {
-      for (unsigned int i = 0; i < m_hlt_counts.size(); ++i) {
-        edm::HLTPathStatus const & path = hltResults.at(i);
-        if (path.index() > m_hltIndices[i].index_l1_seed)
-          m_hlt_counts[i].pass_l1_seed->Fill(lumisection);
-        if  (path.index() > m_hltIndices[i].index_prescale)
-          m_hlt_counts[i].pass_prescale->Fill(lumisection);
-        if (path.accept())
-          m_hlt_counts[i].accept->Fill(lumisection);
-        else if (path.error())
-          m_hlt_counts[i].error ->Fill(lumisection);
-        else
-          m_hlt_counts[i].reject->Fill(lumisection);
-      }
+    if (hltResults.size() == m_hltIndices.size()) {
     } else {
       edm::LogWarning("TriggerRatesMonitor") << "This should never happen: the number of HLT paths has changed since the beginning of the run";
     }
 
-    for (unsigned int i = 0; i < m_datasets.size(); ++i)
-      for (unsigned int j: m_datasets[i])
-        if (hltResults.at(j).accept()) {
-          m_dataset_counts[i]->Fill(lumisection);
+    for (unsigned int d = 0; d < m_datasets.size(); ++d) {
+      for (unsigned int i: m_datasets[d])
+        if (hltResults.at(i).accept()) {
+          m_dataset_counts[d]->Fill(lumisection);
           // ensure each dataset is incremented only once per event
           break;
         }
+      for (unsigned int i = 0; i < m_datasets[d].size(); ++i) {
```

Comment:
ah - ok, I missed the lack of a {} for the loop starting on line 361 (the } on line 366 closes the if statement started on 362)